config/cloud_controller_local_worker_override.yml

```diff
@@ -0,0 +1,3 @@
+# Configurations specific to cc local workers are overridden using this file in capi-release.
```

Comment:
I don't think that you need this empty file. For testing purposes you can simply mock it.

Reply:
This is intended for development purposes only.